internal/config/version/version.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package version
+
+const (
+	Bootloader Type = "bootloader" // Bootloader is the version of the Azure SEVSNP bootloader.
+	TEE        Type = "tee"        // TEE is the version of the Azure SEVSNP TEE.
+	SNP        Type = "snp"        // SNP is the version of the Azure SEVSNP SNP.
+	Microcode  Type = "microcode"  // Microcode is the version of the Azure SEVSNP microcode.
+)
+
+// Type is the type of the version to be requested.
+type Type string
+
+// GetVersion returns the version of the given type.
+func GetVersion(t Type) uint8 {
+	switch t {
+	case Bootloader:
+		return 2
+	case TEE:
+		return 0
+	case SNP:
+		return 6
+	case Microcode:
+		return 93
+	default:
+		panic("invalid version type")
```

Comment:
Should be an error? I guess there will be more errors that should be handled gracefully once this function starts talking to the configapi.